be/src/vec/sink/writer/vtablet_writer.cpp

```diff
@@ -556,8 +556,10 @@ Status VNodeChannel::add_block(vectorized::Block* block, const Payload* payload)
 
 int VNodeChannel::try_send_and_fetch_status(RuntimeState* state,
                                             std::unique_ptr<ThreadPoolToken>& thread_pool_token) {
-    DBUG_EXECUTE_IF("VNodeChannel.try_send_and_fetch_status_full_gc",
-                    { MemInfo::process_full_gc(); });
+    DBUG_EXECUTE_IF("VNodeChannel.try_send_and_fetch_status_full_gc", {
+        MemInfo::process_full_gc(
+                doris::GlobalMemoryArbitrator::process_limit_exceeded_errmsg_str());
```

Comment:
why here need pass the `doris::GlobalMemoryArbitrator::process_limit_exceeded_errmsg_str()` ? if empty() in function internal will call the func.

Reply:
done